RunFullAnalysis.sh

```diff
@@ -16,26 +16,26 @@ ProjectTree=true
 
 #PARAMETERS TO BE SET (use "" for parameters not needed)
 ################################################################################################
-Meson="Dplus" # whether it is Dplus or Ds analysis
-Cent="kpp5TeVFD" # used also to asses prompt or non-prompt and system
+Particle="Lc" # whether it is Dplus,  Ds or Lc analysis
+Cent="kpp13TeVFD" # used also to asses prompt or non-prompt and system
 
-cfgFileData="../AnalysisNonPromptDpp2017/Dplus/configfiles/inputs/config_Dplus_pp_data_tree.yml"
-cfgFileMC="../AnalysisNonPromptDpp2017/Dplus/configfiles/inputs/config_Dplus_pp_MC_tree.yml"
-cfgFileFit="../AnalysisNonPromptDpp2017/Dplus/configfiles/fit/config_Dplus_pp5TeV.yml"
+cfgFileData="configfiles/config_LctopK0s_data_tree.yml"
+cfgFileMC="configfiles/config_LctopK0s_MC_tree.yml"
+cfgFileFit="configfiles/fit/config_Lc_Fit_pp13TeV.yml"
 
-accFileName="accfiles/Acceptance_Toy_DplusKpipi_yfidPtDep_etaDau09_ptDau100_FONLL5ptshape.root"
-predFileName="models/D0DplusDstarPredictions_502TeV_y05_noYShift_all_191017_BDShapeCorrected.root"
+accFileName="accfiles/Acceptance_Toy_LcK0Sp_yfidPtDep_etaDau09_ptDau100_promptD0FONLL13ptshape.root"
+predFileName="models/fonll/feeddown/DmesonLcPredictions_13TeV_y05_FFptDepLHCb_BRpythia8_PDG2020_PromptLcMod.root"
 pprefFileName="" #"ppreference/Ds_ppreference_pp5TeV_noyshift_pt_2_3_4_6_8_12_16_24_36_50.root"
 
-PtWeightsDFileName="systematics/genptshape/ptweights/pp/NonPromptDplusWeights_LHC20a7.root"
-PtWeightsDHistoName="hPtWeightsFONLLDcent"
-PtWeightsBFileName="systematics/genptshape/ptweights/pp/NonPromptDplusWeights_LHC20a7.root"
-PtWeightsBHistoName="hPtWeightsFONLLBcent"
+# PtWeightsDFileName="systematics/genptshape/ptweights/pp/NonPromptDplusWeights_LHC20a7.root"
+# PtWeightsDHistoName="hPtWeightsFONLLDcent"
+# PtWeightsBFileName="systematics/genptshape/ptweights/pp/NonPromptDplusWeights_LHC20a7.root"
+# PtWeightsBHistoName="hPtWeightsFONLLBcent"
```

Comment:
If the code is not working for Lc with these lines, adapt the code. It should be sufficient to leave the variables as `""`